contracts/RiskManagerV1.sol

```diff
@@ -142,6 +161,38 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         earlyCloseAuction(auction);
     }
 
+    /// @notice Begins the collateralization threshold update process.
+    /// @dev Can be called only by the contract owner. The collateralization
+    ///      threshold should be adjusted by taking into account factors like
+    ///      collateralization ratio and gas price.
```

Comment:
Instead of saying 
> ... factors like collateralization ratio and gas price.

I would go a little more specific and say:
> ... factors like, for example, tBTC deposit undercollateralized threshold percent, tBTC deposit severely undercollateralized threshold percent, and the gas price.

Reply:
81c3fe3